src/windows/CameraProxy.js

```diff
@@ -696,21 +696,22 @@ function takePictureFromCameraWindows(successCallback, errorCallback, args) {
     // decide which max pixels should be supported by targetWidth or targetHeight.
     var maxRes = null;
     var UIMaxRes = WMCapture.CameraCaptureUIMaxPhotoResolution;
+    var totalPixels = targetWidth * targetHeight;
     switch (true) {
```

Comment:
didn't notice `switch (true)` was just correcting the case params but wow - haha. definetely turning this to a `if/else`